SmartMenuOptim.Server/wwwroot/css/smo-design-system.css

```
@@ -0,0 +1,166 @@
+    --smo-amber-bright: #F2C572; /* lifted amber — hover, on-dark accent text */
+    --smo-amber-ink: #A9702A;    /* accessible amber-family TEXT on light surfaces
+                                    (raw --smo-amber fails contrast as text) */
+
+    /* ── Brand — neutrals & semantic ─────────────────────────────────
+       Porcelain = the plate/linen: warm off-white, softer than #fff,
+       cuts glare when reading data. */
+    --smo-porcelain: #F5F1E9;  /* text on dark, warm panel bg */
+    --smo-sage: #4F9E7B;       /* success/open/fresh — text-safe on light (canonical) */
+    --smo-sage-soft: #7FB79A;  /* success on DARK surfaces (badges over slate) */
+
+    /* ── Product surfaces (light data UI) ────────────────────────────*/
```

Stylelint (17.14.1)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)